modules/packages/Python.chpl

```diff
@@ -3709,6 +3749,149 @@ module Python {
   proc NoneType.serialize(writer, ref serializer) throws do
     writer.write(this:string);
 
+  //
+  // binary ops
+  //
+  private inline proc _binaryOp(param op: string,
+                                lhs: borrowed Value,
+                                rhs: borrowed Value): owned Value throws {
+    var ctx = chpl_pythonContext.enter();
+    defer ctx.exit();
+    var res = PyObject_CallMethod(
+      lhs.interpreter.operatorModule, op,
+      "OO", lhs.getPyObject(), rhs.getPyObject());
+    lhs.interpreter.checkException();
+    return new Value(lhs.interpreter, res, isOwned=true);
+  }
+  inline proc _binaryOpInPlace(param op: string,
+                                lhs: borrowed Value,
+                                rhs: borrowed Value) throws {
+    var ctx = chpl_pythonContext.enter();
+    defer ctx.exit();
+    var res = PyObject_CallMethod(
+      lhs.interpreter.operatorModule, op,
+      "OO", lhs.getPyObject(), rhs.getPyObject());
+    lhs.interpreter.checkException();
+    lhs.retain(res);
+    lhs.interpreter.checkException();
+  }
+  operator+(lhs: borrowed Value, rhs: borrowed Value): owned Value throws do
```

Comment:
The spacing on this section really made it difficult to follow.  It's nit-picky, but can we have a space between the operator keyword and each operator, and a line break between categories?  E.g.

```Chapel
  operator +=(ref lhs: owned Value, const ref rhs: Value) throws do
    _binaryOpInPlace("iadd", lhs.borrow(), rhs.borrow());

  operator -(lhs: borrowed Value, rhs: borrowed Value): owned Value throws do
    return _binaryOp("sub", lhs, rhs);
  operator -=(ref lhs: owned Value, rhs: borrowed Value) throws do
    _binaryOpInPlace("isub", lhs, rhs);
```

Reply:
I would argue "operator+=" is easier to read than "operator +=", but then we are going to get into a whole code style discussion 😄.  I've updated the code